### ISSUE TITLE
Ruby 3.2 compatibility fix is not yet released

### ISSUE THREAD

**Issue by tomstuart** [libgit2/rugged#943]
#935, which fixes a warning in Ruby 3.2, was merged in July but missed the most recent `rugged` release by two days.

Ruby 3.2 will come out later this month and it’d be great if there was a new version of `rugged` containing this fix so that people upgrading to Ruby 3.2 don’t get the warning.

Could we have a new release between now and Christmas, please?

**Comment by Vasfed** [comment]
ruby 3.2 is released and rugged install fails

**Comment by PikachuEXE** [comment]
@Vasfed Which version did you install and how did you install it?
I install `rugged` 1.5.0.1 and it's fine
If you install 1.0.1 due to `pronto`, the fix is in `master` and not released yet and thus I changed `Gemfile` to use github instead

**Comment by Vasfed** [comment]
@PikachuEXE you're correct, that was 1.0.1, had to switch both gems to use github
```ruby
gem 'rugged', github: 'libgit2/rugged', submodules: true
gem 'pronto', '>= 0.11', github: 'prontolabs/pronto', require: false
```

**Comment by abinoam** [comment]
Hi @Vasfed and @PikachuEXE, 

Pronto has a new release.
https://github.com/prontolabs/pronto/issues/437#issuecomment-1368177742

**Comment by gytisgreitai** [comment]
are there any plans to release this to rubygems?

**Comment by tomstuart** [comment]
[v1.5.1](https://github.com/libgit2/rugged/releases/tag/v1.5.1) was just released, but doesn’t include #935.

**Comment by Aiderlei** [comment]
@tomstuart what is the reason for this? Any updates so far? 

I am getting quite a few warnings:
```
warning: undefining the allocator of T_DATA class Rugged::Repository
warning: undefining the allocator of T_DATA class Rugged::Index
warning: undefining the allocator of T_DATA class Rugged::Patch
```

**Comment by tomstuart** [comment]
No idea, sorry, just summarising what I learned from the commit history. Presumably #935 will be included in the next release, but there hasn’t been one since v1.5.1 in January.

**Comment by fnordfish** [comment]
#935 Seems to be included in v1.6.2: https://my.diffend.io/gems/rugged/1.5.1/1.6.2

Weirdly enough there are no v1.6.0 and v1.6.1.

**Comment by tomstuart** [comment]
This was indeed fixed in v1.6.2.
